setup.py

```diff
@@ -524,15 +543,15 @@ def get_sphinx_theme_version() -> str:
 
 # Dict of all providers which are part of the Apache Airflow repository together with their requirements
 PROVIDERS_REQUIREMENTS: Dict[str, List[str]] = {
-    'airbyte': [],
+    'airbyte': http_provider,
```

Comment:
This is already handled by the cross-provider dep mechanism we have isn't it?

https://github.com/apache/airflow/blob/bcfa0cbbfc941cae705a39cfbdd6330a5ba0578e/airflow/providers/dependencies.json#L2-L4

Reply:
Oh, oh no, not correctly.

That' dependencies.json is only for _optional_ extras, so that would have `apache-airflow-providers-airbyte[http]` -- but that doesn't make sense as it's _required_.

Something to fix in a future PR.

Reply:
Now it's ok - the 'http_provider' dependency is actually dependency to the `apache-airlflow-providers-http` :D